charts/tabulation/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-tabulation-app
+  labels:
+    app: tabulation-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: tabulation-app
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: tabulation-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/tabulation:{{ .Values.image.tag }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: {{ .Chart.Name }}
+              containerPort: {{ .Values.service.port }}
+          env:
+            - name: SMARTY_AUTH_ID
```

Comment:
Similar to the alerting template, I think these env vars need to be updated for the tabulation service. It looks like `fhir_url` and `cred_manager` are the two that the API docs specify, but I wouldn't be surprised if there were others that you could set too.

https://cdcgov.github.io/phdi/latest/containers/tabulation.html#operation/tabulate_endpoint_tabulate_post

Reply:
Added